code/src/index.css

```diff
@@ -1,13 +1,160 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  margin: 0px;
+  background: #e4d3cf;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.input-card {
+  border: 2px solid black;
+  margin:20px auto;
+  width: 300px;
+  height: auto;
+  box-shadow: 5px 5px;
+  padding: 20px;
+  background-color:  #D4D4D4; 
+}
+
+.message-card {
+  border: 2px solid black;
+  margin:20px auto;
+  width: 300px;
+  height: auto;
+  box-shadow: 5px 5px;
+  padding: 20px;
+  background-color: #F2F0F0; 
```

Comment:
Really long words do break out of the message container. I used the css attribute word-break: break-all; to remedy this 😊